webapp/src/Entity/Contest.php

```diff
@@ -634,6 +681,128 @@ public function getProcessBalloons(): bool
         return $this->processBalloons;
     }
 
+    /**
+     * Set processAwards
+     *
+     * @param boolean $processAwards
+     *
+     * @return Contest
+     */
+    public function setProcessAwards($processAwards)
+    {
+        $this->processAwards = $processAwards;
+
+        return $this;
+    }
+
+    /**
+     * Get processAwards
+     *
+     * @return boolean
+     */
+    public function getProcessAwards()
+    {
+        return $this->processAwards;
+    }
+
+    /**
+     * @return Collection|TeamCategory[]
+     */
+    public function getAwardsCategories(): Collection
+    {
+        return $this->awardsCategories;
+    }
+
+    public function addAwardsCategories(TeamCategory $awardsCategory): self
+    {
+        if (!$this->awardsCategories->contains($awardsCategory)) {
+            $this->awardsCategories[] = $awardsCategory;
+        }
+
+        return $this;
+    }
+
+    public function removeAwardsCategories(TeamCategory $awardsCategory): self
+    {
+        if ($this->awardsCategories->contains($awardsCategory)) {
+            $this->awardsCategories->removeElement($awardsCategory);
+        }
+
+        return $this;
+    }
+
+    /**
+     * Set goldAwards
+     *
+     * @param integer $goldAwards
+     *
+     * @return Contest
+     */
+    public function setGoldAwards($goldAwards)
```

Comment:
Same here, we could add `int` as typehint

Reply:
Sure